libr/core/panels.c

```diff
@@ -493,46 +503,48 @@ static void panel_continue(RCore *core) {
 	r_core_cmd (core, "dc", 0);
 }
 
-R_API int r_core_visual_panels(RCore *core) {
-	int okey, key, wheel;
-	int w, h;
-	int asm_comments = 0;
-	int asm_bytes = 0;
-	int have_utf8 = 0;
-	n_panels = 0;
+static void init (RCore *core, int w, int h) {
 	panels = NULL;
+	layout = 0;
 	callgraph = 0;
 	_core = core;
-
-	OS_INIT ();
-	w = r_cons_get_size (&h);
+	menu_pos = 0;
+	menu_x = 0;
+	menu_y = 0;
+	callgraph = 0;
```

Comment:
Duplication - see just a few lines up.

Reply:
.........right i'm sorry i can't believe i missed it. i'd delete that no sooner than i get home...... thank you:)